tests/compat.sh

```diff
@@ -888,7 +888,7 @@ if type lsof >/dev/null 2>/dev/null; then
     }
 else
     wait_server_start() {
-        sleep 1
+        sleep 2
```

Comment:
Is 2 seconds good enough for buildbot? Why not 3 seconds or 5 seconds?

Reply:
The (questionable) reasoning is: slow start issues, though frequent enough to be annoying, are not that systematic, so it could indicate `1` was already nearly enough, so I just doubled it.

Now I'm not opposed to making it large because, now that I think about it, impatient devs should have lsof, so this delay won't annoy them, and for the CI, slower with fewer false positives is probably better than less slow with more false positives, so why not 3 seconds indeed.

Reply:
On developer machines, the `lsof` case will be used. And we're trying to get lsof installed on all CI. Would it be hard to install lsof on all buildbot instances?